extensions/wikia/ARecoveryEngine/ARecoveryUnlockCSS.class.php

```diff
@@ -0,0 +1,91 @@
+<?php
+class ARecoveryUnlockCSS {
+	const CSS_FILE_PATH = 'extensions/wikia/ARecoveryEngine/css/recoveryUnlock.scss';
+	const API_URL = 'http://cs-api.sp-prod.net/';
+	const API_ENDPOINT = 'sp_create_csurl';
+	const CACHE_TTL = 3600 * 10; //10h
+	const TIMEOUT = 10;
+
+	private $outputPage = null;
+
+	public function __construct(OutputPage $outputPage) {
+		$this->outputPage = $outputPage;
+	}
+
+	public function getUnlockCSSUrl() {
+		global $wgServer, $wgSourcePointAccountId, $wgEnableUsingSourcePointProxyForCSS;
+		$wikiaCssUrl = $this->getWikiaUnlockCSSUrl();
+		$memcKey = $wikiaCssUrl;
+		$memCache = F::app()->wg->Memc;
+
+		if ( $wgEnableUsingSourcePointProxyForCSS ) {
+			$jsonData = [
+				"account_id" => $wgSourcePointAccountId,
+				"is_pub_resource" => false,
+				"pub_base" => $wgServer."/__are",
+				"resource" => $wikiaCssUrl
+			];
+
+			$cachedCriptedUrl = $memCache->get( $memcKey );
+			if ( $cachedCriptedUrl ) {
+				return $cachedCriptedUrl;
+			} else {
+				$spQuery = self::postJson(self::API_URL . self::API_ENDPOINT, $jsonData);
```

Comment:
It's being used in `verifyContent()` and `postJson()`. Now I can see it ;) Thanks!